src/jdk.incubator.code/share/classes/jdk/incubator/code/internal/ReflectMethods.java

```diff
@@ -265,55 +260,8 @@ public void visitReference(JCMemberReference tree) {
             JCMethodDecl opMethod = opMethodDecl(lambdaName(), funcOp, codeModelStorageOption);
             classOps.add(opMethod);
             tree.codeModel = opMethod.sym;
-            super.visitReference(tree);
-            if (recvDecl != null) {
-                result = copyReferenceWithReceiverVar(tree, recvDecl);
-            }
-        } else {
-            super.visitReference(tree);
-        }
-    }
-
-    // @@@: Only used for quoted lambda, not quotable ones. Remove?
-    ListBuffer<JCExpression> quotedCapturedArgs(DiagnosticPosition pos, BodyScanner bodyScanner) {
-        ListBuffer<JCExpression> capturedArgs = new ListBuffer<>();
-        for (Symbol capturedSym : bodyScanner.stack.localToOp.keySet()) {
-            if (capturedSym.kind == Kind.VAR) {
-                // captured var
-                VarSymbol var = (VarSymbol)capturedSym;
-                if (var.getConstValue() == null) {
-                    capturedArgs.add(make.at(pos).Ident(capturedSym));
-                }
-            } else {
-                throw new AssertionError("Unexpected captured symbol: " + capturedSym);
-            }
-        }
-        if (capturedArgs.size() < bodyScanner.top.body.entryBlock().parameters().size()) {
-            // needs to capture 'this'
-            capturedArgs.prepend(make.at(pos).This(currentClassSym.type));
         }
-        return capturedArgs;
-    }
-
-    /*
-     * Creates a let expression of the kind:
-     * let $recv in $recv::memberRef
-     *
-     * This is required to make sure that LambdaToMethod doesn't end up emitting the
-     * code for capturing the bound method reference receiver twice.
-     */
-    JCExpression copyReferenceWithReceiverVar(JCMemberReference ref, JCVariableDecl recvDecl) {
```

Comment:
This was used, but I think it was added long ago, before we cleaned up the compiler pipeline as part of flexible constructor bodies.